include/treelite/tree.h

```diff
@@ -77,6 +78,23 @@ class ContiguousArray {
   bool owned_buffer_;
 };
 
+enum class TaskType : uint8_t {
+  kBinaryClfRegr = 0,          // Binary classifier, regressor
```

Comment:
I agree that we should clearly document the meaning of each parameter. Do the descriptions in #228 look reasonable to you?

Reply:
The description for grove-per-class in the "five knobs" section looks reasonable, especially the part describing how the trees are assigned to classes. It would be good to have similar descriptions for other constants.

Reply:
Added detailed description in the docstring.